gatsby-config.js

```diff
@@ -163,12 +163,13 @@ module.exports = {
         sourceMap: true
       }
     },
-    {
-      resolve: `gatsby-plugin-google-analytics`,
-      options: {
-        trackingId: ""
-      }
-    },
+    // TODO: Add google analytics trackingId.
+    // {
+    //   resolve: `gatsby-plugin-google-analytics`,
+    //   options: {
+    //     trackingId: ""
+    //   }
+    // },
```

Comment:
I think we should handle it like this:

```
    {
      resolve: `gatsby-plugin-google-analytics`,
      options: {
        trackingId: process.env.GATSBY_GOOGLE_ANALYTICS_ID
      }
    }
```

The it runs only if the `GATSBY_GOOGLE_ANALYTICS_ID` is set. So we can easily enable and disable in Netlify by setting an environment variable.

I did that on this most recent project and it worked really well.